Pc logic

mariarabin opened the pull request:
Updated the Price Chart logic in the server

mariarabin committed:
1) priceChart.js = removed the test data array
2) data.js = updated the function 'updatingChart(period)'
3) apiRoutes.js = updated the get call for the chart data
4) dataFetch.js = updated the getQuotes function
5) iexinterface.js = updated the intraday function

mariarabin committed:
1) home.handlebars = Updated the 5y id
2) data.js = updated the UpdatingChart function to remove console.log
3) apiRoutes.js = removed console.log
4) Updated dataFetch.js = added different functions per day needed
5) Updated iexInterface = added different functions per day needed

mariarabin committed:
1) data.js = removed the blank array for dataSetsLabel and dataSets

mariarabin committed:
cleanup on removing console.logs

alexjzyang commented on controller/iexInterface.js:
```diff
@@ -64,8 +62,30 @@ iexRequest['income'] = (symbol) => { return axios.get(iexRequest.parseUrl(`stock
+
+iexRequest.test.historicalPrices5y = async () => {
+    const historicalPriceResponse = await axios.get(iexRequest.parseTestUrl(`stock/aapl/chart/5y`));
+    return historicalPriceResponse.data;
```
These 5 functions are exactly the same, could you make use of period parameter and make this a single function please

alexjzyang commented on controller/dataFetch.js:
```diff
@@ -102,18 +102,31 @@ dataFetchManager.getMetrics = async function (metricIds, ticker) {
+    }
+    if (timeRange !== undefined && timeRange.length > 0 && timeRange === "5y") {
+        let response = iexRequest.test.historicalPrices5y()
+        return response;
```
@mariarabin Could you update the iexRequest function to take in period/timerange parameter and return a response? So you don't have 5 identical sets of code, which will be hard to maintain.

alexjzyang replied:
Let me know if you have questions

alexjzyang reviewed (changes requested):
Please see comments, Thanks

mariarabin committed:
Merge branch 'master' into PC-logic

# Conflicts:
#	public/js/data.js

mariarabin committed:
1) dataFetch.js = Updated the getQuotes to be a single function
2) iexInterface.js = Updated the function to be single for historicalPrices
3) data.js = Updated the file to have the setInterval function

ayang726 commented:
Good job!
